cloudtests.py

```diff
@@ -60,18 +60,44 @@ def test_net_name(self):
 
 
 class TestJournalWritten(unittest.TestCase):
+    """
+    Test to check that journal logs get written to disk(/var)
+    and make sure that user-uid.journal is not lost on first boot
+    the test should run on First Boot
+    https://github.com/kushaldas/tunirtests/issues/28
+    https://github.com/kushaldas/tunirtests/issues/48
+    """
 
     def test_journal_written(self):
-        """Test to check if journal gets written to disk"""
 
+        # Find PID
         out, err, eid = system("systemctl show systemd-journald.service -p MainPID")
         out = out.decode('utf-8')
         pid = out[8:-1]
+
+        # Find UID
+        out, err, eid = system("whoami")
+        username = out.decode('utf-8')
+        out, err, eid = system("id -u {0}".format(username))
```

Comment:
if you just run `id -u` with no arguments then you get back the uid without needing to discover the username first.